gollum.json

```diff
@@ -1,11 +1,11 @@
 {
     "Gollum": {
-        "description": "A Git Based Wiki on Alpine. <p>Based on a custom docker image: <a href='https://hub.docker.com/r/adtac/gollum-alpine' target='_blank'>https://hub.docker.com/r/adtac/gollum-alpine</a>, available for amd64 architecture only.</p>",
-        "version": "0.0.2",
+        "description": "A Git Based Wiki on Alpine. <p>Based on the official docker image: <a href='https://hub.docker.com/r/gollumwiki/gollum' target='_blank'>https://hub.docker.com/r/gollumwiki/gollum</a>, available for amd64 and arm64 architecture.</p>",
+        "version": "1.0",
         "website": "https://github.com/gollum/gollum",
         "containers": {
             "gollum-alpine": {
-                "image": "adtac/gollum-alpine",
+                "image": "gollumwiki/gollum",
                 "launch_order": 1,
                 "ports": {
                     "8080": {
```

Comment:
```suggestion
                    "4567": {
```

Reply:
Just a wild guess on this one. Adding as suggestion just in case this is the observed failure.